src/List/ItemList.js

```diff
@@ -52,7 +94,8 @@ const Items = props => {
   };
 
   return (
-    <div className={classes.listInput}>
+    <div>
+      {isOpen && deleteConfirmation}
```

Comment:
I tend to inline my components like this, rather than define them in variables somewhere else:
```js
{isOpen && (
  <Modal></Modal>
)}
```

If the component is _too big_ and bloats this component, I will pull that code out and create a new component:

```js
function ConfirmationModal() {
  return (
    <Modal>
      <p>Delete {itemToDelete}?</p>
      <button onClick={() => handleConfirm()}>OK</button>
      <button onClick={handleCancel}>Cancel</button>
    </Modal>
  );
}
```
I will put that in another file (either a file next to this one, or in another part of the codebase to encourage reuse). One thing I try to _avoid_ is assigning components into variables and using it later like this, because it indirect and harder to read. I try to make my components read from top-to-bottom as much as possible, but this reads bottom-to-top, so I have to backtrack to see what this code does. I hope that makes sense!

Overall, this is fine, but I wanted to share my opinions on semantics.